app/mailers/notification_mailer.rb

```diff
@@ -11,6 +11,12 @@ def daily_summary_email(addressee, summary_report = nil)
     mail(to: addressee.email, subject: t('.subject', date: Date.yesterday))
   end
 
+  def daily_auctions_broadcast_email(addressee, auctions)
+    I18n.locale = addressee.locale
+    @auctions = auctions
+    mail(to: addressee.email, subject: t('.subject'))
```

Comment:
Subject is i18n-enabled by default. How about removing it and rely on automatic look-up?